src/app/content/practiceQuestions/components/QuestionNavigation/index.tsx

```diff
@@ -42,7 +42,11 @@ const QuestionNavigation = ({ question, selectedAnswer, ...props }: QuestionNavi
             onClick={(e: React.MouseEvent) => {
               e.preventDefault();
               dispatch(setAnswer({ answer: null, questionId: question.uid }));
-              dispatch(nextQuestion());
+              if (!isFinalQuestion) {
+                dispatch(nextQuestion());
+              } else {
+                dispatch(finishQuestions());
+              }
```

Comment:
It will be simpler without negation
```suggestion
              if (isFinalQuestion) {
                dispatch(finishQuestions());
              } else {
                dispatch(nextQuestion());
              }
```